third_party/silabs/silabs_board.gni

```diff
@@ -150,6 +156,14 @@ if (silabs_board == "BRD4338A" || silabs_board == "BRD2605A") {
   # ThunderBoards don't have a LCD,
   show_qr_code = false
   disable_lcd = true
+
+  # Custom Board ----------
+} else if (silabs_board == "CUSTOM") {
+  print("Using custom board configuration")
+  print("silabs_family:", silabs_family)
+  print("silabs_mcu:", silabs_mcu)
+  assert(silabs_family != "", "Must specify silabs_family for custom board")
```

Comment:
Nit: We could put the assert at the end of this file.

For any board at this end of the file a family and MCU must be specified

Reply:
Done